class NotFound(Exception):
    pass


class AlreadyExists(Exception):
    pass
